job.go

```diff
@@ -97,3 +161,60 @@ func (jl JobList) Sort(sorter func(i, j int) bool) JobList {
 
 	return jl
 }
+
+//DoesJobContainTaskRange evaluates whether the XML marshalled tasks value contains the regex indicating a sequence of tasks.
+func DoesJobContainTaskRange(j Job) (bool, error) {
```

Comment:
I think this can be simplified to just return a bool by using regex.MustCompile(), which would reduce the downstream error checking burden as well.